doc/manual.rst

```diff
@@ -955,20 +955,43 @@ gray. For example, to highlight ``print "line a"`` and ``print "line b"``:
 
 rst2pdf includes several stylesheets for highlighting code:
 
+* ``abap``
+* ``algol_nu``
+* ``algol``
+* ``arduino``
 * ``autumn``
 * ``borland``
 * ``bw``
 * ``colorful``
+* ``default``
```

Comment:
Is this style picked up as the default colour scheme if the user doesn't explicitly choose another one?

Reply:
Yes, this seems to be our default as well as the pygments default style (which is updated by this PR). I get the same output with and without the "default" stylesheet.